src/driver_zigbee.cpp

```diff
@@ -0,0 +1,469 @@
+        std::snprintf(buf, sizeof buf, "movetosat 0x%02X 0x%04X",
+                      zigbee_msg.zcl_cmd()
+                          .colorcontrol_cmd()
+                          .movetosat_params()
+                          .saturation(),
+                      zigbee_msg.zcl_cmd()
+                          .colorcontrol_cmd()
+                          .movetosat_params()
+                          .transition_time());
+        command += buf;
+      } else if (zigbee_msg.zcl_cmd().colorcontrol_cmd().type() ==
+                 ZigBeeMsg::ZCLCmd::ColorControlCmd::MOVETOHUEANDSAT) {
+        char buf[128];
+        std::snprintf(buf, sizeof buf, "movetohueandsat 0x%02X 0x%02X 0x%04X",
+                      zigbee_msg.zcl_cmd()
+                          .colorcontrol_cmd()
+                          .movetohueandsat_params()
+                          .hue(),
+                      zigbee_msg.zcl_cmd()
+                          .colorcontrol_cmd()
+                          .movetohueandsat_params()
+                          .saturation(),
+                      zigbee_msg.zcl_cmd()
+                          .colorcontrol_cmd()
+                          .movetohueandsat_params()
+                          .transition_time());
+        command += buf;
+      } else {
+        command = "";
+      }
+    } else if (zigbee_msg.zcl_cmd().type() == ZigBeeMsg::ZCLCmd::IDENTIFY) {
+      command += "identify ";
+      if (zigbee_msg.zcl_cmd().identify_cmd().type() ==
+          ZigBeeMsg::ZCLCmd::IdentifyCmd::IDENTIFY_ON) {
+        char buf[128];
+        std::snprintf(
+            buf, sizeof buf, "on 0x%02X 0x%04X",
+            zigbee_msg.zcl_cmd().identify_cmd().identify_on_params().endpoint(),
+            zigbee_msg.zcl_cmd()
+                .identify_cmd()
+                .identify_on_params()
+                .identify_time());
+        command += buf;
+      } else if (zigbee_msg.zcl_cmd().identify_cmd().type() ==
+                 ZigBeeMsg::ZCLCmd::IdentifyCmd::IDENTIFY_OFF) {
+        char buf[128];
+        std::snprintf(buf, sizeof buf, "off 0x%04X", zigbee_msg.zcl_cmd()
+                                                         .identify_cmd()
+                                                         .identify_off_params()
+                                                         .identify_time());
+        command += buf;
+      } else {
+        command = "";
+      }
+    } else {
+      command = "";
+    }
+
+    // Sending the messsage
+    if (tcp_client_.get() != nullptr && command != "") {
+      tcp_client_->Send(command + "\n");
+
+      char buf[128];
+      std::snprintf(buf, sizeof buf, "send 0x%04X 0 0x%02X",
+                    zigbee_msg.zcl_cmd().node_id(),
+                    zigbee_msg.zcl_cmd().endpoint_index());
+      command = buf;
+
+      tcp_client_->Send(command + "\n");
+    }
+
+  } else if (zigbee_msg.type() == ZigBeeMsg::ZLL) {
+  } else if (zigbee_msg.type() == ZigBeeMsg::NETWORK_MGMT) {
```

Comment:
Better,  I send a *Not Implemented* error when the user try to send a ZLL msg. I did this because the ZLL protos are defined in the protobuf repo so a user can actually try to use them.
```
 else if (zigbee_msg.type() == ZigBeeMsg::ZLL) {
    std::cerr << "zzl command not implemented" << std::endl;
    zmq_push_error_->Send("zzl command not implemented");
    return false;
  } else if (zigbee_msg.type() == ZigBeeMsg::NETWORK_MGMT) {
```